python/Scripts/pybind_docs.py

```diff
@@ -0,0 +1,586 @@
+import argparse
```

Comment:
This can be updated / modified as desired. I would like to check it in, even if it's an internal script.